frost-ristretto255/src/lib.rs

```diff
@@ -198,28 +199,51 @@ pub mod keys {
         frost::keys::keygen_with_dealer(num_signers, threshold, &mut rng)
     }
 
+    /// Secret and public key material generated by a dealer performing
+    /// [`keygen_with_dealer`].
     ///
+    /// To derive a FROST(ristretto255, SHA-512) keypair, the receiver of the [`SharePackage`] *must* call
+    /// .into(), which under the hood also performs validation.
```

Comment:
```suggestion
    /// # Safety
    ///
    /// To derive a FROST(ristretto255, SHA-512) keypair, the receiver of the [`SharePackage`] *must* call
    /// .into(), which under the hood also performs validation.
```

Reply:
I don't know if cryptographic/security safety is 'the same' as memory unsafety in the rust doc sense, but it's still about requirements the API user must adhere to:

![Screenshot_20220802-145344.png](https://user-images.githubusercontent.com/552961/182451819-b09cecec-437e-4402-bb17-b2f09545d990.png)

Reply:
Maybe "Security" would be more appropriate?